src/index.ts

```diff
@@ -30,6 +30,27 @@ export default async function caxa({
   );
   const appDirectory = path.join(os.tmpdir(), "caxa", identifier);
   await fs.copy(directory, appDirectory);
+
+  // if we did not copy a node modules folder, but there is a package or package-lock file, run npm install
+  if (!(await fs.pathExists(path.join(appDirectory, "node_modules")))) {
+    const packageLockPath = path.join(appDirectory, "package-lock.json");
+    const packageJsonPath = path.join(appDirectory, "package.json");
+    // prefer using package-lock, if available
+    try {
+      if (await fs.pathExists(packageLockPath)) {
+        await execa("npm", ["ci"], { cwd: appDirectory }).catch();
```

Comment:
```suggestion
        await execa("npm", ["ci"], { cwd: appDirectory, env: { NODE_ENV: "production" } }).catch();
```

Reply:
Also, why did you add a `.catch()` here but not on the `npm i` below?

Reply:
@kylebjordahl Ah, I see! Makes sense. I thought of this as an optimization, but you're right, there are situations in which installing it all is necessary. I'd say to just leave it the way you did.